charts/backingservices/charts/srs/README.md

```diff
@@ -43,8 +43,8 @@ You may enable the component of [Elasticsearch](https://github.com/helm/charts/t
 
 Note: Pega does **not** actively update the elasticsearch dependency in `requirements.yaml`. To leverage SRS, you must do one of the following:
 
-* To use the internally-provided Elasticsearch service in the SRS cluster, use the default `srs.enabled.true` parameter and set the Elasticsearch version by updating the `elasticsearch.imageTag` parameter in the [values.yaml](./values.yaml) to match the `dependencies.version` parameter in the [requirements.yaml](../../requirements.yaml).
-* To use an externally-provided Elasticsearch service with SRS, use the default `srs.enabled.true` parameter, update the `srs.srsStorage.provisionInternalESCluster` parameter in the [values.yaml](./values.yaml) to `false` and then provide connection details as documented below.
+* To use the internally-provided Elasticsearch service in the SRS cluster, use the default `srs.enabled.true` parameter and set the Elasticsearch version by updating the `elasticsearch.imageTag` parameter in the [values.yaml](./values.yaml) to match the `dependencies.version` parameter in the [requirements.yaml](../../requirements.yaml). This method streamlines the deployment process for development and testing environments, but it is not suitable for production environments, which require a fully external Elasticsearch cluster. Additionally, even though you deploy SRS and Elasticsearch together, Pega does not license the Elasticsearch cluster deployed using this method and does not maintain it as part of the Pega Platform support.
```

Comment:
this part is vague and not really client-oriented>
Additionally, even though you deploy SRS and Elasticsearch together, Pega does not license the Elasticsearch cluster deployed using this method and does not maintain it as part of the Pega Platform support.

what do you think of saying it from a client perspective>
Additionally, when  you deploy SRS and Elasticsearch together, the deployment requires a connection to your existing Elasticsearch service maintained by your organization. Your organization must also maintain the license for your Elasticsearch service, since Pega does not include it in the SRS Docker image, nor will Pega provide it as part of Pega Platform container-based deployment support.

Reply:
Hi Taz,
Thanks for your feedback! I'm not sure if your proposed changes are factually correct here: this paragraph is for a method of deploying an open source ES cluster TOGETHER with SRS for ease of deployment. The deployment of ES happens at the same time as SRS, so you do not connect to an already existing ES service, that is the point. The important part is to make it clear that Pega does not license this ES cluster even though the clients use our Helm charts to deploy it, that it is fully open source, and the Helm charts are there only to make the deployment process for testing environments easier. This is not recommended for production, though.

Furthermore, the messaging for this part was actually lifted verbatim from the docs.pega.com articles where I worked with the Search team to convey this in US-558611-1. I'd rather it stayed consistent across platforms.